coreapi/main_api.py

```diff
@@ -86,7 +151,7 @@ def faceexp(cropped_face):
     except Exception as e:
         logger.error(msg=e)
         return {"Error": "Facial Expression Recognition Not Working"}
-    predictions = json.loads(json_response.text)["predictions"]
+    predictions = json.loads(json_response.text).get("predictions", "error")
```

Comment:
instead of error, is there any other more specific value that we  can add?

Reply:
done